client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -399,40 +403,74 @@ class EmailProvidersComparison extends Component {
 
 		const standardPrice = getAnnualPrice( gSuiteProduct?.cost ?? null, currencyCode );
```

Comment:
I think we should compute this price the same way we do for Professional Email when we're showing the standard renewal price to avoid differences between the two options.
```suggestion
		const standardPrice = ! productIsDiscounted && isEligibleForFreeTrial ? formatCurrency( gSuiteProduct?.cost ?? null, currencyCode ) : getAnnualPrice( gSuiteProduct?.cost ?? null, currencyCode );
```

Without this we end up with different formatting for the renewal prices:
<img width="1056" alt="Screenshot 2022-02-23 at 16 15 45" src="https://user-images.githubusercontent.com/3376401/155337845-61d729bc-5262-4a51-b365-fb6787673927.png">